sdk/tests/client/input_selection/delegation_outputs.rs

```diff
@@ -0,0 +1,100 @@
+// Copyright 2024 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use iota_sdk::{
+    client::{api::input_selection::InputSelection, secret::types::InputSigningData},
+    types::block::{
+        address::Address,
+        output::{
+            unlock_condition::AddressUnlockCondition, BasicOutputBuilder, DelegationId, DelegationOutputBuilder,
+            OutputId,
+        },
+        protocol::protocol_parameters,
+        rand::{
+            address::rand_account_address, output::rand_output_metadata_with_id,
+            transaction::rand_transaction_id_with_slot_index,
+        },
+        slot::SlotIndex,
+    },
+};
+use pretty_assertions::assert_eq;
+
+use crate::client::{BECH32_ADDRESS_ED25519_0, SLOT_INDEX};
+
+#[test]
+fn remainder_needed_for_mana() {
```

Comment:
The remainder being generated mana ?

Reply:
in this case it's because of the reward mana